src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_ConditionalExpressions.vb

```diff
@@ -218,6 +218,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return rewrittenRight
             End If
 
+            ' Optimize If(left, right) to left.GetValueOrDefault() when left is T? and right is the default value of T
+            If rewrittenLeft.Type.IsNullableType AndAlso 
+               rewrittenRight.IsDefaultValue AndAlso
```

Comment:
>IsDefaultValue [](start = 30, length = 14)

Is this a function? Even though VB allows to omit an empty argument list, we prefer to not take advantage of that for the benefit of code clarity and readability. Same for IsNullableType above.